documentation/IDTA-01003-a/modules/ROOT/pages/IDTA-01003-a_Specification.adoc

```diff
@@ -59,9 +70,10 @@ Content of data specification template for concept descriptions for properties,
 Note: for details, please refer to [IEC61360-1], property, value_list and term
 ====
 
-
+{aasc3a010}
+////
 +++Constraint AASc-010+++: If _DataSpecificationIec61360/value_ is not empty, _DataSpecificationIec61360/valueList_ shall be empty, and vice versa.
```

Comment:
Should this constraint and the following not be changed according to the naming convention: AASc-3a-010

Reply:
Yes, but this is not the name that will show up in the documentation, this is just a variable for import from constraints.adoc file.

Reply:
Thanks for the clarification!